Backend/dev/migrations/dynamic-offer-driver-app/0164-search-request-and-search-step-tables.sql

```diff
@@ -0,0 +1,2 @@
+CREATE TABLE atlas_driver_offer_bpp.search_try AS TABLE atlas_driver_offer_bpp.search_request;
```

Comment:
Lets add two backfill queries, one backfilling the whole data and other is backfilling the data of last 6 hours so that our db load doesn't exceed threshold while backfilling as for backward compatibility we need last 6 hours data only.